.github/actions/deploy-cluster-run-tests/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Deploy Kind cluster and run tests"
+description: "Restores k8s from the cache previously built, installs kind, builds and runs e2e tests and exports logs and reports"
+runs:
+  using: "composite"
+  steps:
+  - name: Load docker image
+    run: |
+      docker load --input image.tar
```

Comment:
this step depends of a previous step, that creates the `image.tar` having it separate can break easily